docs/meetings.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Weekly Community Meetings
+---
+
+Weekly Meetings
+----------------
+
+### Engineering meeting
+ - Monday at 11:00 - 11:30 AM PST
```

Comment:
Can we replace the date/time with something like [Monday at 11:00AM - 11:30AM PT](https://www.timeanddate.com/worldclock/converter.html?iso=20210125T190000&p1=1440&p2=179&p3=tz_pt). Make it easy for folks not in PT time zone and avoid confusion around Daylight Savings Time